lib/experimental/block-editor-settings-mobile.php

```diff
@@ -33,7 +71,71 @@ function gutenberg_get_block_editor_settings_mobile( $settings ) {
 		$settings['__experimentalEnableListBlockV2'] = true;
 	}
 
-	return $settings;
+	return keep_supported_block_editor_settings_mobile(
+		$settings,
+		array(
```

Comment:
After reviewing the response from the API via `curl`, the values in this array appear to match the network response. I have not tested functionality in the editor, but this list makes sense to me.